test/nools/task-emitter.spec.js

```diff
@@ -287,6 +288,136 @@ describe('task-emitter', () => {
         });
       });
 
+      it('resolvedIf is not required', () => {
```

Comment:
When testing a common unit of logic, this case defaultResolveIf, it good to group it in a describe, it helps to use scoped before and and after hooks.
Sample:

```
describe('defaultResolveIf', () => {
  it('should  ___ when ____', () => {
      ....
  });

  it('should not  ___ when ____', () => {
      ....
  });
});
``